python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -247,6 +265,34 @@ def mcs(self) -> Remote:
     def hexapod(self) -> Remote:
         return self.atcs.rem.athexapod
 
+    @property
+    def temperature(self) -> typing.Optional[float]:
```

Comment:
the `Optional` typing modifier is equivalent to `float | None`, but this only returns `float`. Please remove it...